docs/paper/getting-started/getting-plugins.md

```diff
@@ -0,0 +1,120 @@
+---
+toc_max_heading_level: 4
+---
+
+# Getting Plugins
+
+Plugins are the way to extend the functionality of Paper, beyond the configuration files, and one of
+the most powerful features Paper offers. Functionality added by plugins can range from making milk
+restore hunger or dead bushes grow, to adding completely new and original gamemodes or items.
+
+:::danger Malicious Plugins
+
+Ensure you fully trust the source of any plugin before installing it. Plugins are given **full and
+unrestricted** access to not only your server, but also the machine that it runs on. Because of
+this, it is imperative that plugins be installed only from trusted sources. Be careful!
+
+:::
+
+## Finding Plugins
+
+Before you can install a plugin, you'll need to find what you want to install. Most plugins can be
+found on [SpigotMC](https://www.spigotmc.org/resources/),
+[BukkitDev](https://dev.bukkit.org/bukkit-plugins), or the
+[PaperMC Forums](https://forums.papermc.io/forums/paper-plugin-releases/), while other plugins may
+release on [GitHub](https://github.com). One of the best ways to find plugins isn't to browse any of
+these sites directly, but to search for plugins using a search engine. Searching for the function
+you desire followed by `Minecraft plugin` will often yield good results.
+
+:::tip Spigot and Bukkit Plugins
+
+Paper is compatible with both Spigot and Bukkit plugins. It's okay if a plugin does not specifically
+mention Paper. It'll still work.
+
+:::
+
+## Installing Plugins
+
+1. Once you've found the plugin you'd like to install, download it. Ensure the file you have
+   downloaded ends in `.jar`. Some plugins also distribute as `.zip` files, in which case you will
+   need to extract the file and locate the `.jar` for your platform, often labeled `bukkit` or
+   `paper`.
+2. Once you have the plugin downloaded locally, locate the `plugins` folder from the root directory
+   of your Paper server.
+3. Drag and drop the plugin file (`.jar`) into the `plugins` folder. If you are using a shared
+   hosting service, you may need to use their web panel or SFTP to upload the plugin, however, the
+   procedure will be the same.
+4. Restart your server. The plugin should load.
+5. Check your work. Once the server has finished loading, run the `/plugins` command in-game or type
```

Comment:
add something here about that it should be green, if its read, see troubleshooting